packages/nc-gui/components/smartsheet/Cell.vue

```diff
@@ -139,36 +121,42 @@ const syncAndNavigate = (dir: NavigateDir) => {
 <template>
   <div
     class="nc-cell w-full h-full"
-    :class="{ 'text-blue-600': isPrimary && !virtual && !isForm }"
+    :class="{ 'text-blue-600': isPrimary(column) && !virtual && !isForm }"
```

Comment:
`!virtual` can be removed and not at all available in the context

Reply:
It seems to be used by `Lookup.vue`

```vue
<LazySmartsheetCell :model-value="v" :column="lookupColumn" :edit-enabled="false" :virtual="true" />
```

Is the prop unnecessary then?

Reply:
Ok it's required my bad